apps/rush-lib/src/utilities/VersionControl.ts

```diff
@@ -115,4 +122,37 @@ export class VersionControl {
       .toString();
     return output.trim().split('\n');
   }
+
+  private static _tryFetchRemoteBranch(remoteBranchName: string): boolean {
+    const firstSlashIndex: number = remoteBranchName.indexOf('/');
+    if (firstSlashIndex === -1) {
+      throw new Error(
+        `Unexpected remote branch format: ${remoteBranchName}. ` +
+        'Expected branch to be in the <remote>/<branch name> format.'
+      );
+    }
+
+    const remoteName: string = remoteBranchName.substr(0, firstSlashIndex);
+    const branchName: string = remoteBranchName.substr(firstSlashIndex + 1);
+    const spawnResult: child_process.SpawnSyncReturns<string> = Executable.spawnSync(
+      'git',
+      ['fetch', remoteName, branchName],
+      {
+        stdio: 'ignore'
+      }
+    );
+    return spawnResult.status === 0;
```

Comment:
Please test that `rush change` is still usable when the internet is disconnected.

Reply:
You'll get a warning, but it'll work fine.